common.lua

```diff
@@ -15,7 +15,17 @@ pipeworks.rules_all = {{x=0, y=0, z=1},{x=0, y=0, z=-1},{x=1, y=0, z=0},{x=-1, y
 		{x=0, y=1, z=0}, {x=0, y=-1, z=0}}
 
 pipeworks.mesecons_rules={{x=0,y=0,z=1},{x=0,y=0,z=-1},{x=1,y=0,z=0},{x=-1,y=0,z=0},{x=0,y=1,z=0},{x=0,y=-1,z=0}}
-pipeworks.digilines_rules={{x=0,y=0,z=1},{x=0,y=0,z=-1},{x=1,y=0,z=0},{x=-1,y=0,z=0},{x=0,y=1,z=0},{x=0,y=-1,z=0}}
+
+local digilines_enabled = minetest.get_modpath("digilines") ~= nil
+if digilines_enabled and pipeworks.use_default_digilines_rules then
+	pipeworks.digilines_rules=digilines.rules.default
```

Comment:
```suggestion
	pipeworks.digilines_rules=digiline.rules.default
```
Yes, I know this is legacy version and was already changed in 2017 but it would still be better to go with it unless you'd like to open another PR updating `digiline` to `digilines` first.

Basically for consistency fix all of them or for now go with what is used elsewhere, having both for luacheck isn't acceptable IMO.

Reply:
Where else is digiline used instead of digilines? I found digilines.rules.default in the base digilines mod in presetrules.lua 

I am fine with either as long as it works. I have verified that digiline.rules.default works as well.

Reply:
> Where else is digiline used instead of digilines?

Didn't check where pipeworks is using legacy namespace but I know it is what is currently defined for luacheck, if you're able to change `digiline` to `digilines` here: https://github.com/mt-mods/pipeworks/blob/222a865e17a109980296ac075aeffcfcd976f79f/.luacheckrc#L13 without breaking luacheck then that would be best option but as it is there I believe it is used elsewhere too.

Pipeworks however apparently isn't using current recommended `digilines` namespace, if it would then luacheck would be failing all the time.

Anyway, I did add issue about it so it does not have to be fixed here. It might involve changing multiple files unrelated to this update and better done in separate pull request:
* #143